File: Python_Ruby/OOP_Variable/2.rb
class C
    def initialize(value)
        @value = value
    end

    def setValue(value)
        @value = value
    end

    def getValue()
        return @value
    end
    
    def show()
        p @value
    end
end

c1 = C.new(10)
c1.show()

c1.setValue(20)
p c1.getValue()

File: Python_Ruby/OOP_Variable/2.py
class C(object):
    def __init__(self, value):
        self.value = value

    def setValue(self, value):
        self.value = value

    def getValue(self):
        return self.value
    
    def show(self):
        print(self.value)

c1 = C(10)
print(c1.getValue())

c1.setValue(20)
print(c1.getValue())

c1.show()